server/src/controllers/Events/resolver.ts

```diff
@@ -230,44 +229,36 @@ const getNameForNewAttendance = (event: EventAttendanceName) => {
 
 @Resolver()
 export class EventResolver {
-  @Query(() => PaginatedEventsWithTotal)
+  @Query(() => [PaginatedEventsWithChapters])
```

Comment:
```suggestion
  @Query(() => PaginatedEventsWithChapters)
```
Seems redundant since the array only has a single element.